classes/class-p4-metabox-register.php

```diff
@@ -0,0 +1,625 @@
+<?php
+/**
+ * P4 Metabox Register Class
+ *
+ * @package P4MT
+ */
+
+/**
+ * Class P4_Metabox_Register
+ */
+class P4_Metabox_Register {
+
+	/**
+	 * Meta box prefix
+	 *
+	 * @var string $prefix
+	 */
+	private $prefix = 'p4_';
+
+	/**
+	 * P4_Metabox_Register constructor.
+	 */
+	public function __construct() {
+		$this->hooks();
+	}
+
+	/**
+	 * Class hooks.
+	 */
+	private function hooks() {
+		add_action( 'cmb2_admin_init', [ $this, 'register_p4_meta_box' ] );
+		add_action( 'cmb2_after_form', [ $this, 'cmb2_after_form_do_js_validation' ], 10, 2 );
+	}
+
+	/**
+	 * Register P4 meta box.
+	 */
+	public function register_p4_meta_box() {
+		$this->register_meta_box_header();
+		$this->register_meta_box_post();
+		$this->register_meta_box_open_graph();
+		$this->register_meta_box_campaign();
+	}
+
+	/**
+	 * Register Page Header meta box.
+	 */
+	public function register_meta_box_header() {
```

Comment:
Even though you created the new functions, you forgot to remove their code from the big one, so this one
is still 300 lines long.

Reply:
Ops, It was my miss :( 
Thanks for pointing it out.